packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 48.11.0 - April 16, 2020
+
+## Fixed
+
+#### docs
+
+- Fix broken Java Setup link in [Apex Tests](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/testing) ([PR #2119](https://github.com/forcedotcom/salesforcedx-vscode/pull/2119), [Issue #2108](https://github.com/forcedotcom/salesforcedx-vscode/issues/2108))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Added error message and prompt help for scratch def selection ([PR #2115](https://github.com/forcedotcom/salesforcedx-vscode/pull/2115))
```

Comment:
@rcoringrato-sfdc, please rephrase as:
Added a message to display info when selecting scratch org definition and an error message when no scratch org definition files exist.

Also, I noticed (from the gifs) that the error message has a typo:No scratch **defition** files found. Though not sure if the gifs have the latest message text.

Reply:
@hbc-bindu I update the description, and verified that the error message is correct in the message file - the gif does not reflect the latest message text.